tensorflow_addons/text/parse_time_op.py

```diff
@@ -26,7 +26,7 @@
 tf.no_gradient("Addons>ParseTime")
 
 
-def parse_time(time_string: str, time_format: str, output_unit: str) -> str:
+def parse_time(time_string: str, time_format: str, output_unit: str) -> tf.Tensor:
```

Comment:
The return type is changed. Does it still work for non-eager mode / v1 compat mode?

Reply:
Sorry, I changed the return type because it was wrong in the first place. The function returns a tensor, not a python string.

Reply:
Ah you are right, thanks!